### ISSUE TITLE
[suggestion] ender fluid pipes should be more expensive

### ISSUE THREAD

**Issue by WarlordWossman** [GTNewHorizons/NewHorizons#2997]
#### Which modpack version are you using?
2.0.3.0 should be the same in 2.0.4.0

Ender fluid pipes are quite good, let's check what they currently cost:
![ender_1](https://user-images.githubusercontent.com/16925725/40541507-4c543b4a-601c-11e8-8b99-d164afb0aabc.PNG)
![ender_2](https://user-images.githubusercontent.com/16925725/40541506-4c366b10-601c-11e8-8919-378494eef535.PNG)

So 4000L/s is not the fastest but they instantly distribute to all destinations and also store a lot of fluids at once. You unlock them at HV, and the LCR and distillation tower (2 multiblocks I see them used with the most) are also HV which means you can instantly make them when at that tier.

What I would do is change the liquid to 144mb of molten epoxid or molten PTFE and potentially change the pipe to small or regular titanium fluid pipe.
No need to move the voltage to EV since the assembler would be quite a lot of titanium.
But yeah that way the multi fluid pipes are there at the tier you deal with LCR / distillation tower and the player is actually encouraged to use them instead of ender fluid pipes - which currently you can get instantly when you are at HV.

**Comment by noobyaran9** [comment]
1) They don’t store fluid
2) They’re worse than HV pumps
3) Huge throughput pipes are cheap you can easily get alot
4) You need more than 1 unlike pumps
5) The green alloy is not that cheap and refer to 4
6) Multi fluid pipe is not intuitively easy to use
7) Multi fluid pipe is way cheaper

I suggest move it to EV. The current recipe is good enough.

**Comment by WarlordWossman** [comment]
Meh, polyethylene is used for every single conduit, I think some different fluid would make sense for those.

Also GT pipes are nicer for transporting a single fluid fast but you can't deny that ender fluid conduits are super no brainer when you just connect them to LCR/DT and connect it to a row of super tanks.

**Comment by noobyaran9** [comment]
>Also GT pipes are nicer for transporting a single fluid fast but you can't deny that ender fluid conduits are super no brainer when you just connect them to LCR/DT and connect it to a row of super tanks.

Do you only need one?

**Comment by WarlordWossman** [comment]
You probably need more than 1 but still only a hand full. Vibrant alloy might not be cheap but you want to make some anyways, it's just that ender fluid conduits feel like they should be EV to me, in the end that's obviously an opinion but I had multiple people agree that they are quite cheap for what they do.

**Comment by Dream-Master** [comment]
Sure i can pack all conduits to ev if this help.

**Comment by WarlordWossman** [comment]
not all of them, just ender fluid conduits

**Comment by Methes** [comment]
On the other hand, unlike items, fluids are SUCH a pain to manage, move and use even with AE I'd be sad to see this being nerfed.

**Comment by 0lafe** [comment]
Change vibrant to old recipe, needs tungsten, gl making them in HV in bulk

**Comment by Dream-Master** [comment]
when it was changed ? I not did anything on vibrant alloy. Did gt++ change somethings ?

**Comment by WarlordWossman** [comment]
>On the other hand, unlike items, fluids are SUCH a pain to manage, move and use even with AE I'd be sad to see this being nerfed.
Seriously, minecraft is all about managing items and now that various fluids are a thing you can see how it's seriously lacking in this field.

I requested only 1 tier higher, atm you get ender fluid conduits at the tier you need them which has no progression or reason to use the multifluid pipes.

**Comment by 0lafe** [comment]
In gtnh 1.3 it used to cost tungsten dust instead of the bluesteel/blacksteel thing

**Comment by Dream-Master** [comment]
ok need to check why this was changed

**Comment by WarlordWossman** [comment]
> In gtnh 1.3 it used to cost tungsten dust instead of the bluesteel/blacksteel thing

How do you get tungsten at HV? If you can't get it at all it would make the wither skeleton route even less viable for the nether stars you need for the dire crafting table.

**Comment by 0lafe** [comment]
You can macerate endstone for tungsten, or at least could when that was the recipe for vibrant

**Comment by WarlordWossman** [comment]
if it's possible to get a few ingots of vibrant I am fine with it, I don't think some stuff that depends on vibrant should be gated too much, even if the mixer quest in HV hands out 8 ingots of vibrant alloy, which seems like a bit much if it gets nerfed

**Comment by Methes** [comment]
I tried to kill Ender dragon in HV with nano and jetpack and failed horribly so I personally wouldn't count on getting Endstone in HV.
Having Ender Pipes in EV seems reasonable with multipipes available earlier. Fair enough.

**Comment by Kortako** [comment]
I don't think you should change this at all. Ender Fluid conduits is one of the only big "toys" in HV that made me feel like I'm actually getting rewarded for tiering up. A lot of stuff in this pack is an enormous grind, and I believe you need one or two "really nice toys" every tier to make things enjoyable. In MV and LV managing fluids is a massive pain with GT pipes, and then you unlock EFCs in HV and start feeling like automation of some things can be manageable. You don't use vibrant alloy for many other things, and it's already a pain to make, so I feel like it's use in EFC is perfect and I don't think anything needs to be changed here. If you start tiering everything that makes automation nice behind the higher tiers, mid game won't have any benefits or fun at all. Besides, you already start getting access to AE2 in EV which is way more powerful anyway. Just my 2 cents that I feel quite strongly about lol.

**Comment by richardhendricks** [comment]
@Kortako Did you do MV/LV before multipipes?  I am curious if you tried them out.

I like the current vibrant alloy recipe. Plz don't change to tungsten. :(

**Comment by Kortako** [comment]
@richardhendricks I did try them out. In fact, they were broken as hell and I made the issues to get them fixed way back when. But they're still a bit broken, because the filter shutter didn't work in my testing, and they're also way too expensive to make for what it does. I made an issue about that months ago.

**Comment by WarlordWossman** [comment]
@Methes @Kortako maybe EV is too much, maybe just replace the polyethylene with a plastic that is not super standard and used for everything?
keeping the recipe and changing the fluid to PTFE?

**Comment by Anarack** [comment]
Fluid filters and shutters do work work for the multi pipes. The only issue I knew of for them was when the gt6 pipes in general where having issues, though most people only realize, or read, that fluid filters only block input not output to a pipe, though that should be changed to have both work.

**Comment by Kortako** [comment]
@WarlordWossman I feel like you are really underplaying how expensive it can be with the Vibrant Alloy.  Sure, if you only make 11 conduits for the distillation tower and that's all you ever make, sure it's not that "expensive", but then all it does is make that sorting a LITTLE bit easier (it just saves space).  But I had to make hundreds of them, because my base is a bit spread out, and I need to transport fluids from one side to the other and not lag out my server with 20 different pipelines, and that ended up being even more expensive than a full size distillation tower (or two).  And Vibrant takes even longer to EBF than stainless.

@Anarack Were those fixed?  When I tried them out way back when, the fluid filters would set to a fluid, but still allow any fluid to go through, regardless of what I screwdriver'ed them to.  Either way, there's zero documentation on how they actually work, so having some clarification on how that works would be nice. Here's my original issues on them: #2479 #2448

**Comment by Prewf** [comment]
Please don't forget vacuum freezer recipes were bumped up to be hv-tiered just recently, so that itself adds another pain in the ass to the vibrant alloy chain. Chrome is not readily available until you get to space. Unless you're willing to process tons of redstone.

If you are insistent about changing this recipe, what about swapping polyethylene for PVC? Or what about increasing the number of molten poly you need. Don't change the core recipe itself so drastically, there are OTHER ways of nerfing things besides changing a recipe chain. PTFE and epoxid are completely unnecessary at HV-tier.

**Comment by WarlordWossman** [comment]
They are not meant for long distance, with gt6 pipes you shouldn't lag your base with a pipeline.
Also vibrant alloy is not exactly hard to make imo, but that could just be me. I agree that changing it to tungsten would be too much, nobody does the dragon at HV to mine 3,5 stacks of endstone for 2 dusts on average.

@Prewf got a point with the VF, what I mean is just that everything is kinda renewable. I gues PVC would already be a bit better than polyethylene but PTFE would make them more of an item you can't just spam. Not too sure here.

**Comment by Kortako** [comment]
@WarlordWossman "They're not meant for long distance" Uh, excuse me? So now you are dictating how I should play?  If I need to long distance my fluids, that's my decision.  I have a chemical plant/refinery that's 150 blocks away from my main base set up, and that's done for my own aesthetic and gameplay reasons.  They work fine for long distance, so saying they're not "meant" for it is completely disingenuous.

And it's not just "a pipeline", it would be like 20 pipelines for me.  All of them 150 blocks long.  Making 20 sets of GT pipes isn't a big deal material wise, it's just a crap-ton of polyethylene.  But setting all that up is mindless and there isn't anything novel and "smart" about that.  EFCs save that space and effort, at a much higher cost (vibrant).  I still don't see your reasoning on how they are so OP?

And what you replied to Prewf: Once again, I don't see how EFCs are renewable?  Vibrant isn't renewable?  Chrome?  You can't "just spam them", it's a huge cost.  I'm telling you from first hand experience.

**Comment by Anarack** [comment]
Ok after messing with them some more, they do filter fluids correctly... as long as there is more than one fluid in the pipe, if there is less they ignore fluid filters.

**Comment by Anarack** [comment]
Then if you do not want to do that cost, why not look into other options such as carts and fluid unloaders, which can filter, any of the wireless item or fluid transfer methodes.

**Comment by WarlordWossman** [comment]
>"They're not meant for long distance" Uh, excuse me? So now you are dictating how I should play? If I need to long distance my fluids, that's my decision. I have a chemical plant/refinery that's 150 blocks away from my main base set up, and that's done for my own aesthetic and gameplay reasons. They work fine for long distance, so saying they're not "meant" for it is completely disingenuous.

What kind of logic is that? Balance can't cater to everyone, if my gameplay desicion would be to use nether star fluid pipes in steam age do we now have to make that possible? Complete nonsense and not an argument at all.
The defining feature of them is that they deal with multiple fluids, and that's not something you have to do on long range. I you choose to make a base that needs 150 ender fluid conduits that's your price to pay, as easy as that.

Chrome is just a bunch of redstone, you would wonder how fast you get a stack with some redlons.

@Prewf probably just perspective then because I am currently at HV, it's just what I said, with distillation tower and LCR in HV why would you ever use the multi fluid pipes? Maybe if you can't wait for your vacuum freezer, but that's not the biggest argument in the world imo.
There is literally tons of options esp with item conduits that transport cells everywhere.

**Comment by richardhendricks** [comment]
@Prewf See, you are making an argument to change this to PTFE.  Saying "I only had one use for PTFE!" to me says we need more PTFE uses.

And it is far from "switching out PE for some random plastic".  PE is used for tons of shit already. SMD circuits, boards, pipes, etc.  Something else for PTFE is good.

It doesn't have to stay at 144mB.  18 or 36 mB PTFE per 1 EFC would be fine, and not onerous.  By HV you should have found Mica, so you can save two stacks and convert the rest to Fluorine for PTFE.  And GT++ is adding fluorite ore.

You still have the option of using the GT multipipes, and if there is a bug we can fix it.

With the amount of flexibility these pipes provide, there is a good argument for making the pipes *much* harder.  Like adding a circuit requirement to each.

**Comment by Dream-Master** [comment]
so what are the discussion rn? put it to Ev ? change polyethylene to something else? change vibrant alloy to tungsten need as source? I know we discussed it here but at the end of this I need something everyone van live with. Thanks

**Comment by Kortako** [comment]
>  if my gameplay desicion would be to use nether star fluid pipes in steam age do we now have to make that possible

That's not what I'm saying at all, what kind of a comparison is that?  You're trying to make me sound like I want to make it **cheaper**, when I'm really saying **I think it's already expensive enough**.  My base is something that exists and is sensible, and your theoretical one is something no one would do.  It's a complete strawman argument. 
This is my base, you've probably seen it before:
![2018-01-20_14 11 21](https://user-images.githubusercontent.com/5300243/40565518-c72437da-6021-11e8-91d4-6a868a052123.png)
My chemical plant is that building on the left in the back with the truck in front of it, and my main base is the construction yard in the front.  It doesn't look that far, but actually it turns out to be about 150 blocks. 

"Balance" should also take into consideration the use case of EFCs.  It isn't something you just make one of, like a single machine, it's something you need to expand.  Pipes aren't exhorbitantly expensive because you need to make a lot of them.  They only get more expensive as the flow rate goes up, and as has already been said, the flow rate of EFCs aren't that great.

You also are leaving out the stainless costs, which are 3 stainless per pipe, because they are normal pipes.  And chrome is just a bunch of redstone in the way that Titanium is just a bunch of rutile.  It's a huge PITA to process, so really you only want to process ruby ore if possible. And I don't know about you, but I don't find redstone veins that often, so when I finally went to the moon and got chromite, it was a godsend.

As for Redlon, not everyone goes deep into Thaumcraft, but if you do, so what?  Technically, Tin Copper Iron and Gold are renewable with IC2 crops, should everything that uses that be nerfed too?  See, I can strawman too.  I'm trying hard to not be offensive, but you calling my arguments "nonsense" kind of set me off.

TLDR: I think I can get behind PTFE IF it is less than 144mb, as moron suggested.  Or PVC at 144mb as an alternative.

**Comment by Dream-Master** [comment]
ok so we stay at hv let vibrant like it is and change only the fluid and amount?

**Comment by richardhendricks** [comment]
OK, so to both increase and reduce pain, here is a proposed recipe

1 tiny PTFE pipe
1 stainless steel plate
36 mB of liquid vibrant alloy

This makes the EFCs significantly cheaper for SS and VA, but adds a new use for PTFE.  1 tiny pipe is only 1/2 ingot of PTFE, so should not be too onerous.  A single Mica vein should allow for plenty of EFCs even for large bases, esp. since you only need to connect 1 pipe point-to-point.

And now there is a good option for a player getting a distillation tower - pipe it up with multipipes now, or get PTFE infrastructure in place to do EFCs.

@Kortako This should still allow you to use it for long distance.

**Comment by Kortako** [comment]
@richardhendricks Cool, I can get behind that.

**Comment by WarlordWossman** [comment]
Discussed with moron a bit to meet somewhere in the middle, the idea in the end is that currently you never have to get out of your way to make the ender fluid conduits, with a tiny bit of PTFE it introduces a decision you have to make - just like deciding if you wanna save metal plates by using polyethylene for machine hulls in LV.
With that decision involved, the multi fluid pipes will be the go to option while ender fluid conduits will be there if you set up PTFE.

@Prewf a so called "random plastic" gets less random if you add more uses for it since you all of a sudden have multiple reasons to get it ;)

**Comment by richardhendricks** [comment]
WAT? You can use PE for machine hulls?!

TIL

**Comment by Prewf** [comment]
I dunno, I view PTFE as "high" tier plastic. It essentially replaces PE to make hulls later. So to see it being used so early on was why I was so against it (vs other non PE plastics as an alternative). That being said, this compromise sounds good :)

Also @richardhendricks what? you've been shape crafting your hulls this whole time or something? Gotta revisit those old inefficient recipes man.

**Comment by richardhendricks** [comment]
:ribbon: of shame. :)

**Comment by CalterOK** [comment]
In my opinion there is no reason to change something in this recipe.

**Comment by Dream-Master** [comment]
Stainless steel small pipe is replaces with a PTFE Tiny Pipe

**Comment by Prewf** [comment]
@Dream-Master 
recipe should be this:

**1 tiny PTFE pipe
1 stainless steel plate
36 mB of liquid vibrant alloy**

not:
1 tiny PTFE pipe
1 molten PE
1 vibrant alloy plate

**Comment by Dream-Master** [comment]
No I not reduce vibrant alloy. We nerf not buff it.

**Comment by Methes** [comment]
Rant!
Good job turning the pack into Clicker Heroes!
You click (craft yourself to death) and wait (everything takes SO long).
Oh and endless NEI searching, forgot about that.
/Rant!

**Comment by Dream-Master** [comment]
It wasn't my issue. The recipe can be reverted if this feel better.  @Methes

**Comment by WarlordWossman** [comment]
No revert here, 0,5 PTFE vs 3 stainless steel, the stainless would have actually taken you a longer time.
I wouldn't say it's cheaper but makes you set up something which is good.
If we revert it to the old recipe it's the same old waiting game for the EBF again, I would rather encourage people to build up different setups.
